apps/api/src/app/integrations/dtos/get-active-integration-response.dto.ts

```diff
@@ -1,9 +1,8 @@
-import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
-import { ChannelTypeEnum } from '@novu/shared';
-import { CredentialsDto } from './credentials.dto';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+
 import { IntegrationResponseDto } from './integration-response.dto';
 
 export class GetActiveIntegrationResponseDto extends IntegrationResponseDto {
-  @ApiProperty()
-  selected: boolean;
+  @ApiPropertyOptional()
+  selected?: boolean;
```

Comment:
the selected flag is kind of optional for now because the endpoint behaves in the old way and in a new returning this additional flag... after we remove ff this will be always defined